swapserverrpc/staticaddr.proto

```diff
@@ -135,6 +135,15 @@ message ServerStaticAddressLoopInRequest {
     // swap payment. If the timeout is reached the swap will be aborted on the
     // server side and the client can retry the swap with different parameters.
     uint32 payment_timeout_seconds = 8;
+
+    /*
+    The optional swap amount the client is attempting to swap. If specified the
+    server will take out this amount from the total value of provided
+    deposit_outpoints and will send the change back to the static address. If
+    not specified the server will use the total amount of the deposit_outpoints
+    as swap amount.
```

Comment:
What happens, if the amount is too close to the total amount (so the change would be dust)? I think it makes sense if it fails and suggests to remove the flag to use the whole amount.
Could you add the actual expected behavior in this case to the comment, please?

Also, if the amount is equal to the total amount, it makes sense just to succeed, right? And if higher - fail.

Reply:
I expanded the comment to explain. 

Note, there is no flag to use the whole amount of provided deposits for backwards compatibility. If deposits are specified without an amount the whole amount is used.